ContactViewPage: do not hardcode the ContactManager name

mardy opened the pull request:
Use whatever name is defined by the AddressBook components. This means
that if we change the backend in the AddressBook, the Dialer will
automatically use the correct backend.

Part of https://github.com/ubports/ubuntu-touch/issues/997

mardy committed:
ContactViewPage: do not hardcode the ContactManager name

Use whatever name is defined by the AddressBook components. This means
that if we change the backend in the AddressBook, the Dialer will
automatically use the correct backend.

lduboeuf commented:
@mardy there is still those 2 places where "galera" is used:
https://github.com/ubports/dialer-app/blob/xenial/src/qml/DialerPage/ContactDialPadSearch.qml#L76
and https://github.com/ubports/dialer-app/blob/xenial/src/contactutils.h#L28

mardy committed:
ContactUtils: remove default value for ContactManager name

This function is always called with the first argumetn set, so there's
no need to have a default parameter.

mardy commented:
> @mardy there is still those 2 places where "galera" is used: https://github.com/ubports/dialer-app/blob/xenial/src/qml/DialerPage/ContactDialPadSearch.qml#L76 and https://github.com/ubports/dialer-app/blob/xenial/src/contactutils.h#L28

You are right, it's all because I checked out the dialer-app repo on gitlab, and it didn't have these occurrencies. I've updated the branch, please have another look. :-)
